packages/react-events/src/__tests__/Focus-test.internal.js

```diff
@@ -327,3 +327,65 @@ describe('Focus event responder', () => {
     expect(Focus.displayName).toBe('Focus');
   });
 });
+
+describe('Focus event responder - Mac', () => {
```

Comment:
You shouldn't need to create another top-level `describe`, setup the feature flags, etc. Put the test into the relevant `onFocus` test block that already exists above.

Reply:
Totally agree, I wasn't sure how to approach mocking navigator for this single test. The problem is that setup for each test is done in beforeEach and ideally I would like to mock this BEFORE common beforeEach, but this is not quite possible - or I don't know how to do it, but after searching through jest documentation I couldn't find a way to do this. Any ideas?

Reply:
Oh because you need the `navigator` mocked before `Focus` is evaluated? Hmm, not sure. I'll look into it

Reply:
yeah, that's exactly the "problem" with putting this into existing describe block

Reply:
Have a look at how it's being done here: https://github.com/facebook/react/pull/15761. Once you reuse that pattern for this PR we can merge it. Thanks!

Reply:
See https://github.com/facebook/react/blob/0f7cc2ba845e34d7a312819e27bbfa88cdb7d49f/packages/react-events/src/__tests__/Press-test.internal.js#L39-L46

Reply:
Thanks for the tip! Done.